testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/versions/KotlinGradlePluginVersions.groovy

```diff
@@ -100,6 +100,10 @@ class KotlinGradlePluginVersions {
         kgpVersion < VersionNumber.parse(latestStable)
     }
 
+    List<String> languageVersionsSupportedByLatestStableOrRc() {
+        return LANGUAGE_VERSIONS.drop(2)
```

Comment:
❌ that's confusing. Why 2? When will it be changed? etc.

Reply:
I clarified this in https://github.com/gradle/gradle/pull/31378/commits/16625563ad944b20c47cdf24b0b1dcf55bdca257